modules/402-ingress-nginx/templates/failover/daemonset.yaml

```diff
@@ -52,6 +52,8 @@ spec:
       name: {{ $crd.name }}
   template:
     metadata:
+      annotations:
+        {{ include "helm_lib_ds_eviction_annotation" . | nindent 8 }}
```

Comment:
may be func name should be more readable - something like helm_lib_prevent_ds_eviction_annotation ?